src/data.ts

```diff
@@ -92,6 +92,7 @@ const countries: CountryInterface[] = [
   { name: 'Eritrea', dial_code: '+291', code: 'ER', flag: '🇪🇷' },
   { name: 'Estonia', dial_code: '+372', code: 'EE', flag: '🇪🇪' },
   { name: 'Ethiopia', dial_code: '+251', code: 'ET', flag: '🇪🇹', local_name: 'ኢትዮጵያ' },
+  { name: 'Eswatini', dial_code: '+268', code: 'SZ', flag: '🇸🇿' },
```

Comment:
Could you reorder this please?
I think `Eswatini` should come before `Ethiopia`